mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -841,6 +841,18 @@ be from 0 to 999,999,999 inclusive.`,
 				Description: `A set of key/value label pairs to assign to instances created from this template,`,
 			},
 
+			"resource_policies": {
+				Type:             schema.TypeList,
+				Optional:         true,
+				ForceNew:         true,
+				MaxItems:         1,
```

Comment:
`MaxItems: 1` can be removed from the field in Terraform, in case more resource policies will be supported later. Do you know if the service side has the validation?

Reply:
Yes, there is validation in Service. Also there is no plans to support more resource policies.

But I would prefer to keep to be consistent: resource policies in disk has this limitation and also Instance resource has this limit: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/resources/resource_compute_instance.go.erb#L889